fhir-query-helper-base/src/main/java/io/elimu/a2d2/cds/fhir/helper/QueryingServerHelperBase.java

```diff
@@ -717,6 +717,17 @@ protected static void iRestfulClientFactoryConfig(IRestfulClientFactory factory)
 
 	public abstract FhirResponse<IBaseResource> fetchServer(final String resourceType, String resourceQuery);
 	
+	public FhirFuture<FhirResponse<IBaseResource>> fetchDirectAsync(String resourceType, String id, String asyncId) {
+		Callable<FhirResponse<IBaseResource>> callable = new Callable<FhirResponse<IBaseResource>>() {
+			@Override
+			public FhirResponse<IBaseResource> call() throws Exception {
+				String resourceQuery = fhirUrl + "/" + resourceType + "/" + id;
```

Comment:
Should this just call `getResourceByIdResponse` (line 455) with parameter usePath=true?